packages/shared/src/utils/ssr.ts

```diff
@@ -1,3 +1,3 @@
 export const inClientSide = (): boolean => {
-  return typeof window !== 'undefined' && typeof document !== 'undefined';
+  return typeof window !== 'undefined';
```

Comment:
- ❓@nikosdouvlis @dimkl didn't we recently update this to include `document` in the check ?

Reply:
In Expo window is defined but document is not, so this check doesn't work.